src/jagadeeshMJun20/ExtractItemId.java

```diff
@@ -0,0 +1,38 @@
+/*Extract ItemId from below inpu9t
+String input = https://www.facebook.com/technocredits/temId=12345/
+String input = https://www.facebook.com/technocredits/empid=123&itemId=12345/
+String input = https://www.facebook.com/technocredits/itemId=12345&empid=123/
+String input = https://www.facebook.com/technocredits/empid=123/
+Output : 
+item id is 12345 
+        OR
+no item id present. */
+package jagadeeshMJun20;
+ class ExtractItemID {
+	void displayItemID(String input) {
+		String itemId = "";
+		if (input.contains("itemId")) {
+			String temp1 = input.substring(input.indexOf("itemId"));
+			for (int index = 0; index < temp1.length(); index++) {
+				if (temp1.charAt(index) >='0' && temp1.charAt(index) <='9') {
+					itemId += temp1.charAt(index);
+					if (!(temp1.charAt(index + 1) >='0' && temp1.charAt(index+1) <='9'))
```

Comment:
This if is not required, you should write else of parent of this block.

if (input.contains("itemId=")) {
			String str = input.split("itemId=")[1];
			String output = "";
			for (int index = 0; index < str.length(); index++) {
				if (Character.isDigit(str.charAt(index))) {
					output += str.charAt(index);
				} else {
					break;
				}
			}
			System.out.println("Item id is " + output);
		} else {
			System.out.println("No itemID present.");
		}

Suggesting an alternative You can also write : 
try{
String str = input.split("itemId=")[1].replace("/", "").split("&")[0];
System.out.println("Item id is " + str);

}catch(StringIndexOutOfBoundsException siob){
			System.out.println("Item ID -> Not present");
		}